v3/sockjs/options.go

```diff
@@ -80,6 +89,7 @@ var DefaultOptions = Options{
 	DisconnectDelay:   5 * time.Second,
 	ResponseLimit:     128 * 1024,
 	WebsocketUpgrader: &websocket.Upgrader{},
+	AllowedMethods:    []ReceiverType{},
```

Comment:
having slice here might be confusing, it would allow multiple occurence of the same value.

Reply:
I know :x 
But how bad is the impact of having multiple/duplicate entries? 
if it was added once, does not matter how many times it is added again, it will only have one entry on the handler.mapping

On handler.go, I create a map and ignore all of those duplicate entries.
```
	// using map to reduce AllowedMethods to uniq keys
	var indexedAllowedMethods map[ReceiverType]bool
	for _, rtype := range h.options.AllowedMethods {
	    indexedAllowedMethods[rtype] = true
	}
```

 
 The other solutions would be using `map[ReceiverType]bool`, where empty is all, and boolean can be used as the flag for adding or not